examples/phi3/pass_configs/data_configs.json

```diff
@@ -14,9 +14,7 @@
         },
         "pre_process_data_config": {
             "params": {
-                "corpus_strategy": "join",
                 "text_template": "### Question: {prompt} \n### Answer: {response}",
```

Comment:
The terminating ',' at the end of the line makes this a non-compliant json.

Reply:
Removed.